docs/PORTING.md

```diff
@@ -117,12 +117,18 @@ int     flash_area_write(const struct flash_area *, uint32_t off,
 int     flash_area_erase(const struct flash_area *, uint32_t off, uint32_t len);
 /*< Returns this `flash_area`s alignment */
 uint8_t flash_area_align(const struct flash_area *);
-/*< Initializes an array of flash_area elements for the slot's sectors */
-int     flash_area_to_sectors(int idx, int *cnt, struct flash_area *ret);
+/*< What is value is read from erased flash bytes. */
+uint8_t flash_area_erased_val(const struct flash_area *);
+/*< Reads len bytes from off, and checks if the read data is erased. */
+int     flash_area_read_is_empty(const struct flash_area *fa, uint32_t off,
+                     void *dst, uint32_t len);
+/*< Given flash area ID, return info about sectors within the area. */
+int     flash_area_get_sectors(int fa_id, uint32_t *count,
+                     struct flash_sector *sectors);
 /*< Returns the `fa_id` for slot, where slot is 0 (primary) or 1 (secondary) */
-int     flash_area_id_from_image_slot(int slot);
+int     flash_area_id_from_multi_image_slot(int image_index, int slot);
 /*< Returns the slot, for the `fa_id` supplied */
```

Comment:
Comment should have `area_id` (betteer) or parameter should be renamed `fa_id`. Please mention `image_index` in the comment.